libraries/botbuilder/tests/teamsHelpers.test.js

```diff
@@ -195,4 +196,35 @@ describe('TeamsActivityHelpers method', function () {
             assert.strictEqual(channelId, '');
         });
     });
+
+    describe('teamsGetTeamOnBehalfOf()', function () {
+        it('should return onBehalfOf list', async function () {
+            const activity = {
+                channelData: {
+                    onBehalfOf: [
+                        {
+                            itemId: 0,
+                            displayName: 'onBehalfOfTest',
+                            mentionType: 'person',
+                            mri: 'mriTest',
+                        },
+                    ],
+                },
+            };
+            const onBehalfOf = teamsGetTeamOnBehalfOf(activity)[0];
+            assert.strictEqual(onBehalfOf.displayName, 'onBehalfOfTest');
```

Comment:
I think the 'expected' value goes first, and we can use the const defined previously instead of a hardcoded value:
```suggestion
            assert.strictEqual(activity.channelData.onBehalfOf.displayName, onBehalfOf.displayName);
```